explore.js

```diff
@@ -82,6 +82,12 @@
 
         } else if (messageData.action == "goToUrl") {
             Window.location = messageData.visit;
+        } else if (messageData.action == "navigateBack") {
+            location.goBack();
+        } else if (messageData.action == "navigateHome") {
+            location.goToLocalSandbox();
```

Comment:
should get home bookmark address from "LocationBookmarks.getHomeLocationAddress" and if none is set then go to the sandbox,